### PR TITLE
fix(find-foreman): fix path finding algorithm in foreman-find

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: packages/find-foreman/index.js [hunk 1]
```diff
@@ -16,18 +16,24 @@ const foremanLocation = (throwError = true) => {
...
-    if (fs.existsSync(result)) fullPath = result;
+    return fs.existsSync(result);
   });
 
-  if (!fullPath && throwError) {
+  if (!relativePath && throwError) {
     throw new Error(notFound);
   }
+  if (!relativePath) {
+    if (throwError) {
```
Comment (laviro) [inline packages/find-foreman/index.js:29]: should we remove lines 25-27 ?

**Comment by ShimShtein** [comment]
@sharvit, done and removed redundant `if`

**Comment by sharvit** [comment]
To give a context @ShimShtein have to following setup:
- /foreman/foreman/package.json
- /foreman/my-plugin/package.json

The find-foreman script returned `/foreman` instead of `/foreman/foreman` because the `foreEach` loop didn't stop once found a result.

I am wondering if we should allow consumers to provide their own path to foreman for more complicated setups (env variable?).

@johnpmitsch, for you to follow :)

Merging...

**Pull request merged**